lua/harpoon/test/list_spec.lua

```diff
@@ -22,11 +22,16 @@ describe("list", function()
         })
         local list_config = Config.get_config(config, "foo")
 
-        local list = List.decode(list_config, "foo", { "foo:bar", "baz:qux" })
+        local list = List.decode(list_config, "foo", {
+            "foo:bar",
+            nil,
+            [3] = "baz:qux",
+        })
         local displayed = list:display()
 
         eq(displayed, {
             "foo---bar",
+            "",
```

Comment:
maybe deserialize `nil` to something other than the empry string? I didn't put much thought into it